charts/dev/livebook/templates/_secrets.tpl

```diff
@@ -0,0 +1,14 @@
+{{/* Define the secrets */}}
+{{- define "livebook.secrets" -}}
+{{- $secretName := (printf "%s-livebook-secrets" (include "tc.v1.common.lib.chart.names.fullname" $)) }}
+{{- $secretKeyBase := randAlphaNum 48 | b64enc -}}
```

Comment:
Just to make sure, 
- this will be b64 encoded.
- when stored to the secret will also be b64 encoded
- when its injected to the env will be the value of the first encoding.

eg `abcd` -> `jiberish_v1`

secret will hold `jiberish_v2` 
container will see `jiberish_v1`

if the container should see `abcd`, the `b64enc` should be removed. other wise ignore this... jiberish!

Reply:
I admit that is confusing. But that's what I've intended. From the [Livebook documentation](https://github.com/livebook-dev/livebook/?tab=readme-ov-file#environment-variables) about this variable:

> Must be at least 64 characters long and it can be generated by commands such as: `openssl rand -base64 48`

Reply:
Cool! I've seen one or two other apps that need the same, lets add a comment that the b64enc is intended there and lets merge this!

Reply:
I mean... `abcd` is a random value. So one could call it `jiberish_v0` 🤪